### DISCUSSION TITLE
posts/2022/05/mulling-over-migration/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#10] [Announcements]
# posts/2022/05/mulling-over-migration/

Some tips about moving your website to Astro.

https://www.brycewray.com/posts/2022/05/mulling-over-migration/

**Comment by davidsneighbour** [comment]
I think any serial-migrator should have knowledge of [Remark](https://github.com/remarkjs/remark), which is basically a Markdown parser, linter, anything tool. Then write simple Markdown with your own "shortcode" markup and for any system based requirements would  write a conversion of the Markup into "their" format. This will make it very easy to prepare any content for specific requirements.

**Reply by brycewray** [reply]
Fair enough, although I think I addressed that in the related footnote.
